yellowbrick/text/postag.py

```diff
@@ -408,7 +540,6 @@ def postag(
         from most to least frequent.
     kwargs : dict
         Pass any additional keyword arguments to the PosTagVisualizer.
-
```

Comment:
Since we're at the bottom of the code - just a gentle reminder to modify your editor settings so lines aren't removed by accident!